docs/pages/index.mdx

```diff
@@ -67,22 +55,18 @@ Get started with Teleport Access:
 - [Issue and govern identities for automated
   systems](enroll-resources/machine-id/introduction.mdx) using Machine ID.
```

Comment:
I believe this belongs under Machine & Workload Identity, now. We could also add a bullet there linking to Workload Identity, as well, so this isn't a single isolated bullet point.

Reply:
First pass at a workload identity bullet point (for the section below). @mmcallister @eglenn-teleport @thedevelopnik please feel free to revise the content; I just didn't want to suggest adding content without providing something to start with:
`- Replace long-lived API keys and environment secrets with short-lived cryptographic identities using x.509 certificates or JWTs using Workload Identity.`

I'll probably mess it up if I try to insert the link, but I'd link to the Workload Identity intro to match how we're handling the Machine Identity link: https://goteleport.com/docs/enroll-resources/workload-identity/introduction/

Reply:
That wording works for me right now.